docs/site/Validation-REST-layer.md

```diff
@@ -0,0 +1,163 @@
+---
+lang: en
+title: 'Validation in REST Layer'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation-REST-layer.html
+---
+
+At the REST layer, there are 2 types of validation:
+
+1. [Type validation](#type-validation)
+2. [Validation against OpenAPI schema specification](#validation-against-openapi-schema-specification)
+
+## Type Validation
+
+The type validation in the REST layer comes out of the box in LoopBack.
+
+> Validations are applied on the parameters and the request body data. They also
+> use OpenAPI specification as the reference to infer the validation rules.
+
+Take the `capacity` property in the `CoffeeShop` model as an example, it is a
+number. When creating a `CoffeeShop` by calling /POST, if a string is specified
+for the `capacity` property as below:
+
+```json
+{
+  "city": "Toronto",
+  "phoneNum": "416-111-1111",
+  "capacity": "100"
+}
+```
+
+a "request body is invalid" error is expected:
+
+```json
+{
+  "error": {
+    "statusCode": 422,
+    "name": "UnprocessableEntityError",
+    "message": "The request body is invalid. See error object `details` property for more info.",
+    "code": "VALIDATION_FAILED",
+    "details": [
+      {
+        "path": ".capacity",
+        "code": "type",
+        "message": "should be number",
+        "info": {
+          "type": "number"
+        }
+      }
+    ]
+  }
+}
+```
+
+## Validation against OpenAPI Schema Specification
+
+For validation against an OpenAPI schema specification, the
+[AJV module](https://github.com/epoberezkin/ajv) can be used to validate data
```

Comment:
> can be used to 

`is used to ` 

(If we are doing this automatically for the developer, let's use `is used to`. If the developer needs to program this, let's use `can be used to` .